app/assets/stylesheets/components/comments.scss

```diff
@@ -0,0 +1,31 @@
+@import '../config/import';
+
+.crayons-comment {
+  margin-bottom: $su-4;
+  font-size: $fs-s;
+
+  &__meta {
+    display: flex;
+    align-items: center;
+
+    a {
+      display: flex;
+      align-items: center;
+    }
+  }
+
+  &__body {
+    padding-left: $su-7;
+
+    a:hover {
+      text-decoration: underline; //todo: default for all in-content links
+    }
+
+    p {
+      display: -webkit-box;
+      -webkit-line-clamp: 2;
+      -webkit-box-orient: vertical;
+      overflow: hidden;
+    }
```

Comment:
This is how I'm clipping the comments to two lines..everything I found online says it will work across browsers! Thoughts @ludwiczakpawel?

Reply:
I've seen on screenshot that you have `...` at the end of truncated comment... Is this done via backend? If not, you may want to add this:

```
text-overflow: ellipsis
```

it should automatically add `...` at the end of 2nd line.

Reply:
No, it's wild: the `-webkit-line-clamp` property adds the ellipsis when `overflow` is set to `hidden`!